src/pages/sns.tsx

```diff
@@ -0,0 +1,480 @@
+            <motion.div variants={transitions.item} className="flex-1">
+              <motion.h2
+                className="tw-heading-5 md:tw-heading-3 mb-10 md:mb-16"
+                variants={transitions.item}
+              >
+                What an SNS DAO is <span className="text-razzmatazz">NOT</span>
+              </motion.h2>
+              <div className="md:w-8/10">
+                <h3 className="tw-heading-6 md:tw-heading-5 mb-4">
+                  Not an ICO
+                </h3>
+                <p className="tw-paragraph mb-10">
+                  No funds are handed over to developers. The creation of an SNS
+                  is initiated by a decentralized entity (
+                  <Link
+                    className="text-infinite underline hover:no-underline hover:text-black"
+                    href="https://nns.ic0.app/"
+                  >
+                    the NNS
+                  </Link>
+                  ), and ICP raised by the decentralization sale is allocated to
+                  the treasury of the newly born SNS DAO.
+                </p>
+                <h3 className="tw-heading-6 md:tw-heading-5 mb-4">
+                  Not an IPO
+                </h3>
+                <p className="tw-paragraph">
+                  This is not a liquidity exit or a public listing. It’s the
+                  start of a new journey where you as the initial creator and
+                  your users have the opportunity to form a digital community,{" "}
+                  <strong>100% on-chain</strong>, to co-evolve and grow the
+                  platform or application.
+                </p>
+              </div>
+            </motion.div>
+            <motion.div variants={transitions.item} className="flex-1">
+              <motion.h2
+                className="tw-heading-5 md:tw-heading-3 mb-10 md:mb-16"
+                variants={transitions.item}
+              >
+                What an SNS DAO <span className="text-blue">IS</span>
+              </motion.h2>
+              <div
+                className="
+                  md:w-8/10
+                  prose
+                  prose-h3:tw-heading-6 prose-h3:md:tw-heading-5 prose-h3:mb-4
+                  prose-p:tw-paragraph prose-p:mb-3
+                  prose-ul:mb-4 prose-ul:list-none prose-ul:pl-0 prose-ul:tw-paragraph
+                  prose-li:bg-[url('/img/checkmark.svg')] prose-li:bg-no-repeat prose-li:bg-left-top prose-li:pl-8 prose-li:my-3 prose-li:leading-6
+                  "
+              >
+                <h3>Decentralization Sale</h3>
+                <p>
+                  Funds are put in a treasury, a portion of which is allocated
+                  to the founders in the form of locked neurons for developing a
+                  service. Launching an SNS DAO requires the following:
+                </p>
+                <ul>
+                  <li>An existing dapp</li>
```

Comment:
I think technically this is not a requirement. 
You could launch an SNS without a dapp and let the SNS create the dapp.
Do we still want to phrase it like this as we think this will be the prevalent way people will use it?

Reply:
I think we should keep it as such - otherwise it's too close to an ICO. We certainly don't want to promote empty SNSs with great promise coupled with zero dissolve delay proposals. WDYT?

Reply:
This is fine for me. Resolving this thread. Thank you.